staging/src/k8s.io/kube-scheduler/config/v1beta3/types.go

```diff
@@ -89,6 +89,11 @@ type KubeSchedulerConfiguration struct {
 	// with the extender. These extenders are shared by all scheduler profiles.
 	// +listType=set
 	Extenders []Extender `json:"extenders,omitempty"`
+
+	// MaintainCacheNonLeader enables caching for non-leader candidates. This makes non-leader candidates
+	// start with watching cache. Doing so will have faster failover at cost of higher memory overhead.
+	// Defaults to true.
+	MaintainCacheNonLeader *bool `json:"maintainCacheNonLeader,omitempty"`
```

Comment:
I think we should only populate this field to v1, but we should still have conversion codes for v1beta2 and v1beta3, cc @alculquicondor for confirmation.

Reply:
yeah, only in v1 is enough.

Reply:
Not sure if I have made it right. PTAL!